src/data_handling/data_handler.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
+
+from src.data_handling.dataloader import DataLoader
+
+
+class DataHandler:
```

Comment:
Csináljunk egy `DataInterface` osztályt, amit példányosít a `DataHandler` és minden tagváltozót ebbe a `DataInterface`-be tegyünk át. Így az adattárolót, amit adogatunk tovább leválasztjuk az adattagokat kiszámító metódusokról, tehát tovább mindig CSAK adatot adunk, de függvényeket nem (ez memória spórolás, hogy a függvényeknek létrehozott memória helyek nem lesznek többszörösen tárolva).